src/sentry/static/sentry/less/includes/bootstrap/theme.less

```diff
@@ -182,7 +178,7 @@
   #gradient >
     .vertical(
       @start-color: lighten(@navbar-inverse-bg, 10%) ; @end-color: @navbar-inverse-bg
-    );
+    );;
```

Comment:
this shouldn't have 2x `;` but I think this was inserted by your auto formatter

Reply:
Grrr thanks for catching this!